_episodes/02-installation.md

```diff
@@ -135,10 +139,10 @@ Next, download the file
 [`julia-1.0.5-linux-x86_64.tar.gz`](https://julialang-s3.julialang.org/bin/linux/x64/1.0/julia-1.0.5-linux-x86_64.tar.gz)
 by clicking the link or going to the [Julia downloads
 page](https://julialang.org/downloads/). Download the file directly to the
```

Comment:
Please remove this paragraph, no need to have three different instructions for downloading the file.